src/_includes/assets/css/screen.css

```
@@ -2470,3 +2470,7 @@ a.banner:hover span {
   margin-right: 0;
   margin-left: 19px;
 }
+
+.rtl-layout .post-full-content p code {
+    word-break: break-word; 
```

This seems to be the fallback for `word-break: break-word` now that `break-word` is a deprecated value:

```suggestion
  word-break: normal;
```